source/faucet/main.d

```diff
@@ -84,145 +86,172 @@ private struct State
     }
 }
 
-/*******************************************************************************
+public class Faucet : IFaucet
+{
+    /// Config instance
+    private Config config;
 
-    Splits the Outputs from `utxo_rng` towards `count` random keys
+    /// State instance
```

Comment:
A brief description of what the state represents and when it may change would be nice.

Reply:
Also a default initializer here would help make it clear what the initial state is.

Reply:
> A brief description of what the state represents and when it may change would be nice.

Should it be `///` or `/**` style ?

Reply:
We usually use `///` for fields. You can have multiple lines of them, that's okay.